presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -429,9 +431,9 @@ class SystemConfig : public ConfigBase {
 
   int32_t concurrentLifespansPerTask() const;
 
-  int32_t httpExecThreads() const;
+  int32_t numHttpIoThreadsHwMultiplier() const;
```

Comment:
Do we want to use a double so we can configure it to a fraction of hw concurrency? Thanks!

Reply:
Maybe this is not a bad idea.
I thought about it before, but not this time.